iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/comparison_ops.mlir

```diff
@@ -54,3 +54,34 @@ vm.module @module {
 }
 
 // -----
+
+vm.module @module {
+  // CHECK-LABEL: @module_cmp_eq_ref
+  vm.func @cmp_eq_ref(%arg0 : !vm.ref<?>, %arg1 : !vm.ref<?>) -> i32 {
+    // CHECK-NEXT: %0 = emitc.call "vm_cmp_eq_ref"(%arg3, %arg4) : (!emitc.ptr<!emitc.opaque<"iree_vm_ref_t">>, !emitc.ptr<!emitc.opaque<"iree_vm_ref_t">>) -> i32
+    %0 = vm.cmp.eq.ref %arg0, %arg1 : !vm.ref<?>
```

Comment:
Why are these changing from `%arg0, %arg1` to `%arg3, %arg4`? Are new arguments getting added?

(I see that the tests above also have the same behavior)

Reply:
> Are new arguments getting added?

Yes, that is exactly the case here. Since you suggested to insert some inline comments, this is something that @henri-gruender could document with a revised patch :)

Reply:
Will do!

Reply:
The best place to comment on this will be a test for the `vm.func` conversion, that I will add in a future pull request.